File: java/sui-user-transaction/src/main/java/ru/smsoft/sui/usertransactionconfigurier/autoconfigure/AutoConfiguration.java
package ru.smsoft.sui.usertransactionconfigurier.autoconfigure;

import org.springframework.context.annotation.ComponentScan;
import org.springframework.core.Ordered;
import org.springframework.transaction.annotation.EnableTransactionManagement;

@ComponentScan("ru.smsoft.sui.usertransactionconfigurier")
@EnableTransactionManagement(order = Ordered.LOWEST_PRECEDENCE - 1)
public class AutoConfiguration {

}

File: java/sui-backend/src/main/java/ru/smsoft/sui/suibackend/config/Autoconfiguration.kt
package ru.smsoft.sui.suibackend.config

import org.springframework.context.annotation.ComponentScan
import org.springframework.context.annotation.Configuration
import org.springframework.core.Ordered
import org.springframework.transaction.annotation.EnableTransactionManagement

@Configuration
@EnableTransactionManagement(order = Ordered.LOWEST_PRECEDENCE - 1)
@ComponentScan("ru.smsoft.sui.suibackend")
class Autoconfiguration
